dbaas/account/models.py

```diff
@@ -26,6 +26,18 @@ class Meta:
         verbose_name_plural = _("users")
         verbose_name = _("user")
 
+    @property
+    def external(self):
+        can_be_external = False
+        for team in self.team_set.all():
```

Comment:
Maybe u can all this code for: self.team_set.filter(team__external=True).exists()